tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -215,6 +216,9 @@ public static int setDateOrder(int dateOrder) {
     SHUFFLE_HASH_APPENDER_PAGE_VOLUME("tajo.shuffle.hash.appender.page.volumn-mb", 30),
     HASH_SHUFFLE_PARENT_DIRS("tajo.hash.shuffle.parent.dirs.count", 10),
 
+    // Final output Configuration --------------------------------------------------
+    FINAL_OUTPUT_FILE_FORMAT("tajo.final.output.file-format", BuiltinStorages.TEXT, Validators.javaString()),
```

Comment:
Each key level should be meaningful, and left keys should be more general than those of right ones. 
I'd like to recommend ``DEFAULT_OUTPUT_FILE_FORMAT`` and ``tajo.output.file-format``.